src/components/expand-viewer/expand-viewer.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { ITheme } from '../../theme/types';
+
+export const expandViewButtonStyle = (theme: ITheme, isHovered: boolean) => ({
+  padding: '4px 12px',
+  borderRadius: '50px',
+  borderWidth: '1px',
+  borderStyle: 'solid',
+  /**
+   * since it's not possible to use :hover selector in inline React style I had to handle it manually,
+   * there could be added a css style which avoided since there is no css files in the project
+   */
```

Comment:
is it? We have  plenty of usages of `:hover` in the project

e.g. `src/components/row/group/row-group.styles.ts`

Reply:
yeah I see, since @giardiv told me to not go with rebass stuff I tried making my own button component, but I think we should still keep with rebass until we pick a new stack

Reply:
yeah, just use rebass stuff until we get rid off it. We just don't know when that happens yet, as it's a very big change to be made
until that, rebass is doing its job